### ISSUE TITLE
Interactive mode should only ask to install packages after all files have been processed

### ISSUE THREAD

**Issue by HitkoDev** [raineorshine/npm-check-updates#1160]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates`
- [x] I am using `node >= 14`

The new prompt https://github.com/raineorshine/npm-check-updates/commit/a35404256d4782b404ef3f1847818f7058a7c8e6 keeps asking to install packages for each file, and it's rather annoying when dealing with multiple packages. There should be a way to ask only once at the end or to turn this off completely.

---

## Steps to Reproduce

.ncurc:

```js
{
    "packageFile": [
        "package.json",
        "packages/*/package.json"
    ]
}
```

**Comment by raineorshine** [comment]
Yes, one prompt at the end would be much better.

I have implemented a fix in 109236c and published to `latest`.

**Comment by HitkoDev** [comment]
Thank you, works great!
